broker-node/config/app.php

```diff
@@ -177,6 +177,7 @@
         // App\Providers\BroadcastServiceProvider::class,
         App\Providers\EventServiceProvider::class,
         App\Providers\RouteServiceProvider::class,
+        AltThree\Segment\SegmentServiceProvider::class
```

Comment:
Nit: move this to line `171` since its' a package service provider.

Do you still need to call `Segment::init("SrQ0wxvc7jp2XDjZiEJTrkLAo4FC2XdD");` with this and the `SEGMENT_WRITE_KEY` ENV variable added? I think the framework should have taken care of it?

Reply:
I thought so too.